osler/inventory/test_views.py

```diff
@@ -0,0 +1,127 @@
+from django.test import TestCase
+from django.urls import reverse, resolve
+from osler.inventory.models import DrugCategory, MeasuringUnit, Manufacturer, Drug
+from osler.inventory import views
+from .tests import drug_dict
+from osler.core.tests.test_views import log_in_provider, build_provider
+
+class TestDrugList(TestCase):
+
+    fixtures = ['core', 'workup','inventory']
+
+    def setUp(self):
+        log_in_provider(self.client, build_provider())
+        self.diff_drug = {
+                            'name': 'Differentdrug',
+                            'unit': MeasuringUnit.objects.last().pk,
+                            'dose': 500.0,
+                            'stock': 5,
+                            'expiration_date': '2100-01-01',
+                            'lot_number': 'HGFEDCBA',
+                            'category': DrugCategory.objects.last().pk,
+                            'manufacturer': Manufacturer.objects.last().pk
+                        }
+
+    def test_drug_list_view(self):
+        drug = Drug.objects.create(**drug_dict())
```

Comment:
if you're going to make the same drug in each test, just make in the setUp

Reply:
Fair point!